prov/rxm/src/rxm_ep.c

```diff
@@ -1138,7 +1138,7 @@ static int rxm_ep_msg_cq_open(struct rxm_ep *rxm_ep)
 		return ret;
 	}
 
-	if (cq_attr.wait_obj != FI_WAIT_FD)
+	if ((cq_attr.wait_obj != FI_WAIT_FD))
```

Comment:
nitpick: don't need double parentheses.

Reply:
Bad cleanup from removing the debug code I had there from before.  Will update